citation.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.0.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Clark
+    given-names: Andrew
+  - family-names: Foster
+    given-names: Michael
+    orcid: https://orcid.org/0000-0001-8233-9873
+  - family-names: Somers
+    given-names: Richard
+    orcid: https://orcid.org/0000-0002-1101-9722
+  - family-names: Turner
+    given-names: Robert
+  - family-names: Walkinshaw
+    given-names: Neil
+  - family-names: Wild
+    given-names: Christopher
+title: "Causal Testing Framework"
+version: 3.1.0
```

Comment:
@cwild-UoS can this (and below) be automatically updated somehow?

Reply:
After a fair bit of digging around I can't find any tools or workflows to currently do this. I've found a good discussion on this in [another project](https://github.com/xarray-contrib/cf-xarray/issues/398).

Reply:
It's tricky. Might need to be done by hand for now. Similar problem for ORDA releases.